content/components/selectpanel.mdx

```diff
@@ -10,19 +10,12 @@ import ComponentLayout from '~/src/layouts/component-layout'
 export default ComponentLayout
 import {AccessibilityLink} from '~/src/components/accessibility-link'
 
-<Note>
-  This component is currently under development in React and Rails and might not have all its functionality yet.
-  Internally this epic can be tracked <a href="https://github.com/github/primer/issues/2396">here</a>.
-</Note>
-
 ## Overview
 
 The select panel allows you to select one or multiple items from a list, and it can be enhanced with additional controls like a search input to expedite the selection process. It is often used to augment information of a wide variety of items.
 
 For instance, you may encounter a select panel when you need to assign someone to a task or add labels to an issue or pull request. It can also be utilized to select a specific branch in a code repository.
 
-It's important to understand the differences between the select panel and [action menu](/components/action-menu). The primary distinction lies in the fact that an action menu lacks the ability to include additional form controls and automatically closes after making a selection. Therefore, if you require a filterable list or the capability to make multiple selections at once, the select panel is the appropriate choice.
```

Comment:
Kinda wish we still had this information. A lot of people do get confused about this.

Reply:
`the capability to make multiple selections` is kinda wrong here as menu's support this too.

Reply:
Incorporated it in the first paragraph now.